lesson_4_js_basics.js

```diff
@@ -0,0 +1,69 @@
+/* 1. Написати функцію, яка у випадку якщо типи двох аргументів string поверне їх зконкатенований вираз,
+якщо типи двох аргументів number - поверне їх помножене значення
+та у будь якому випадку не співпадіння поверне текст There are no type matches for operation */
+
+function typeChecker(arg1, arg2) {
+let result;
+if(typeof arg1 === 'string' && typeof arg2 === 'string'){
+result = arg1 + arg2;
+}
+else if(typeof arg1 === 'number' && typeof arg2 === 'number'){
+result = arg1 * arg2;
+}
+else { result = 'There are no type matches for operation';
+}
+ 
+ return result
+}
+
+// Приклад роботи функції:
+console.log(typeChecker(2, 3)); // виведе 6
+console.log(typeChecker("My", "text")); // виведе Mytext
+console.log(typeChecker(true, 5)); // виведе There are no type matches for operation
+
+/* 2. Написати функцію, яка, використовуючи тернарний оператор, виводить у консоль Yes, якщо число
+позитивне, та No якщо воно негативне */
+function isNegative(value) {
+let result = value > 0 ? "Yes" : "No"
+return result
+}
+
+// Приклад роботи функції:
+isNegative(-1); // виведе у консоль "Yes"
+isNegative(1); // виведе у консоль "No"
+
+/* 3. Написати функцію, яка повертає true якщо перший аргумент ділиться на другий без остачі, 
+або якщо ні - повертає false */
+function divider(num1, num2) {
+    let result;
+    if(num1 % num2 === 0){
+    result = true;
+    }
+    else {
+    result = false;
+    }
+     
+     return result
+}
+
+// Приклад роботи функції:
+console.log(divider(4, 2)); // виведе у консоль true
+console.log(divider(3, 2)); // виведе у консоль false
+
+/* 4. Написати функцію, яка рекурсивно виводить в консоль аргумент та збільшує його на 2 якщо аргумент не
+перевищує за значенням число 10 */
+function recursiveAddition(value) {
+if(value >= 10){
```

Comment:
```suggestion
if(value > 10){
```
10 не виводилася в твоєму варіанті. За це зніму бал